src/helix-ui/elements/HXElement.less

```diff
@@ -1,12 +1,11 @@
 @import (reference) "vars";
 
-*, *::before, *::after {
-  box-sizing: border-box;
-  color: inherit;
-  font: inherit;
-  letter-spacing: inherit;
-}
-
-input::-ms-clear {
```

Comment:
This selector/definition was moved to `HXSearchElement.less`, because it's the only place that needs it right now. In the future, if we need to use it in multiple files, we'll need to convert it into a mixin.